knative-operator/deploy/resources/kourier/kourier-release-0.17.yaml

```diff
@@ -6,73 +6,6 @@ metadata:
     networking.knative.dev/ingress-provider: kourier
 ---
 apiVersion: v1
-kind: ConfigMap
-metadata:
```

Comment:
@aliok look, Ali!  Removal of things we like to remove as well :joy_cat:

Reply:
LOL :laughing: 

I think my idea of having a unit test that checks the manifest is useful in this case too.

These kind removals can be forgotten (happened to us)